src/client/controller/home.js

```diff
@@ -290,13 +295,17 @@ module.controller('HomeCtrl', ['$rootScope', '$scope', '$document', '$HUB', '$RP
             };
 
             getUser().then(function () {
-                getOrgs().then(function () {
-                    getLinkedOrgs();
-                    getRepos();
+                $scope.isLoading = true;
+                $q.all([
```

Comment:
I made a little small refactor here to:
1. getOrgs() is in line with getRepos() by putting getLinkedOrgs in it.
2. Use q.all() to get orgs, repos and gists parallelly.

Reply:
That looks and works much better now 👍  Thanks!